packages/umi/src/plugins/umiAlias.ts

```diff
@@ -2,7 +2,7 @@ import { IApi } from '@umijs/types';
 
 export default (api: IApi) => {
   api.chainWebpack(memo => {
-    memo.resolve.alias.set('umi', process.env.UMI_DIR);
+    memo.resolve.alias.set('umi', process.env.UMI_DIR ?? '');
```

Comment:
这不需要用空值判断，要么是 undefined ，要么是 string

```suggestion
    memo.resolve.alias.set('umi', process.env.UMI_DIR || '');
```

Reply:
好像在这个场景下 `??` 和 `||` 没有区别，结果都是一样的 🤔

Reply:
嗯嗯，解下冲突

Reply:
不用了，已经在 master 里解决了，close 了